packages/lib/survey/service.ts

```diff
@@ -375,8 +401,10 @@ export async function updateSurvey(updatedSurvey: TSurvey): Promise<TSurvey> {
       attributeFilters: updatedSurvey.attributeFilters ? updatedSurvey.attributeFilters : [], // Include attributeFilters from updatedSurvey
     };
 
-    revalidateTag(getSurveysCacheTag(modifiedSurvey.environmentId));
-    revalidateTag(getSurveyCacheTag(modifiedSurvey.id));
+    surveyCache.revalidate({
```

Comment:
when a survey gets updated, we don't revalidate the cache for `getSurveysByAttributeClassId` and `getSurveysByActionClassId`. But we should :-) Luckily we should have all the information in the method :-)

Reply:
Done